packages/yoastseo/README.md

```diff
@@ -58,10 +60,7 @@ const url = "https://my-site-url.com/path-to-webworker-script.js"
 const worker = new AnalysisWorkerWrapper( createWorker( url ) );
 
```

Comment:
Is using the `createWorker` here still relevant and is it still working? I see that in `content-analysis-webworker` (and `content-analysis`) app, we use the `workerUnwrapped` instead. Is `workerUnwrapped` method more preferrable in this case? 🤔

Reply:
It works inside `yoastseo`, but not really outside -- I'll change it to just use `new Worker( url )` as [that should be supported in every browser](https://caniuse.com/webworkers) 👍